packages/cli/src/lib/cli.ts

```diff
@@ -28,7 +28,7 @@ export const cli = (args: string[]) =>
         'Run collect skiping the coverage plugin, other plugins from config file will be included.',
       ],
       [
-        'code-pushup upload --persist.outputDir=dist --persist.filename=cp-report --upload.apiKey=$CP_API_KEY',
+        'code-pushup upload --persist.outputDir=dist --upload.apiKey=$CP_API_KEY',
         'Upload dist/cp-report.json to portal using API key from environment variable',
```

Comment:
The description is now inaccurate, it would upload `dist/report.json` if the default filename is used.

Reply:
fixed